### PR TITLE
hotfix remove dom from children Fragment #2

### PR DESCRIPTION
**Pull request by lloydzhou**

### CODE REVIEW

**Comment by luwes** [comment]
thanks for the contribution! 

one test seems to be failing, could you look into that? 
thanks

File: little-vdom.js [hunk 1]
```diff
@@ -144,14 +144,17 @@ const diffChildren = (parentDom, newChildren, oldVNode) => {
...
 
   return oldVNode;
 };
 
+function removePatchedChildren(child) {
+  const { _children = [], _normalizedChildren=[], _patched } = child
+  // remove children
+  _children.concat(_patched).map(c => c && removePatchedChildren(c))
+  // remove dom from _normalizedChildren or itself
+  _normalizedChildren.concat(child).map(i => i && i.dom && i.dom.remove())
```
Comment (lloydzhou) [inline little-vdom.js]: @luwes 
this line remove _normalizedChildren first, so remove text node first.
image
after remove dom in _normalizedChildren, then `child.dom.outerHTML = "<li></li>"` not `" <li>d</li>"`

Comment (lloydzhou) [reply little-vdom.js]: may be just remove child.dom is ok?

Comment (lloydzhou) [reply little-vdom.js]: remove `li`, then `li.text` removed

**Comment by lloydzhou** [comment]
@luwes 
fixed the test case.

**Comment by luwes** [comment]
thank you ❤️

**Pull request merged**